backend/src/main/resources/db/migration/V0013__ healthcare_cover.sql

```diff
@@ -0,0 +1,6 @@
+ALTER TABLE person
+  ADD COLUMN healthcare_cover VARCHAR;
```

Comment:
Be careful, if you rename the field to `healthCareCoverage`, this will also have to be renamed to `health_care_coverage`.